src/NUnitFramework/framework/Internal/Tests/TestSuite.cs

```diff
@@ -174,7 +176,7 @@ public override int TestCaseCount
         /// <summary>
         /// The arguments to use in creating the fixture
         /// </summary>
-        public object[] Arguments { get; internal set; }
+        public override object[] Arguments { get { return arguments; } }
```

Comment:
I don't understand the change away from the automatic property, which works fine and is the way we generally do things nowadays.